apps/dashboard/src/components/primitives/input.tsx

```diff
@@ -102,7 +102,7 @@ export const inputVariants = tv({
       sm: {
         root: 'rounded-lg',
         wrapper: 'gap-2 px-2.5',
-        input: 'h-9 text-paragraph-xs',
+        input: 'h-[2.35rem] text-paragraph-xs',
```

Comment:
This aligns our input with the editor's size for single line. It's easier to do the adjustment here.